source/agora/consensus/EnrollmentManager.d

```diff
@@ -137,8 +137,8 @@ public class EnrollmentManager
         static ubyte[] buffer;
 
         // check validity of the enrollment data
-        if (auto reason = isInvalidEnrollmentReason(this.ledger.getBlockHeight() + 1,
-            enroll, this.utxo_set.getUTXOFinder()))
+        if (auto reason = isInvalidEnrollmentReason(enroll,
+            this.ledger.getBlockHeight() + 1, this.utxo_set.getUTXOFinder()))
         {
             this.logMessage("Invalid enrollment data, Reason: " ~ reason,
```

Comment:
This part is never entered in the unittest. There needs to be a test with invalid enrollment too.

Reply:
Wait, wrong line!

Reply:
It's just UFCS from the old code. and I also tried to insert an invalid test code.